pkg/util/paramtable/quota_param.go

```diff
@@ -1915,6 +1921,60 @@ but the rate will not be lower than minRateRatio * dmlRate.`,
 	}
 	p.L0SegmentRowCountHighWaterLevel.Init(base.mgr)
 
+	p.DeleteBufferRowCountProtectionEnabled = ParamItem{
+		Key:          "quotaAndLimits.limitWriting.deleteBufferRowCountProtection.enabled",
+		Version:      "2.4.11",
+		DefaultValue: "false",
+		Doc:          "switch to enable delete buffer row count quota",
+		Export:       true,
+	}
+	p.DeleteBufferRowCountProtectionEnabled.Init(base.mgr)
+
+	p.DeleteBufferRowCountLowWaterLevel = ParamItem{
+		Key:          "quotaAndLimits.limitWriting.deleteBufferRowCountProtection.lowWaterLevel",
+		Version:      "2.4.11",
+		DefaultValue: "32768",
+		Doc:          "delete buffer row count quota, low water level",
+		Export:       true,
+	}
+	p.DeleteBufferRowCountLowWaterLevel.Init(base.mgr)
+
+	p.DeleteBufferRowCountHighWaterLevel = ParamItem{
+		Key:          "quotaAndLimits.limitWriting.deleteBufferRowCountProtection.highWaterLevel",
+		Version:      "2.4.11",
+		DefaultValue: "65536",
+		Doc:          "delete buffer row count quota, high water level",
+		Export:       true,
+	}
+	p.DeleteBufferRowCountHighWaterLevel.Init(base.mgr)
+
+	p.DeleteBufferSizeProtectionEnabled = ParamItem{
+		Key:          "quotaAndLimits.limitWriting.deleteBufferSizeProtection.enabled",
+		Version:      "2.4.11",
+		DefaultValue: "false",
+		Doc:          "switch to enable delete buffer size quota",
+		Export:       true,
+	}
+	p.DeleteBufferSizeProtectionEnabled.Init(base.mgr)
+
+	p.DeleteBufferSizeLowWaterLevel = ParamItem{
+		Key:          "quotaAndLimits.limitWriting.deleteBufferSizeProtection.lowWaterLevel",
+		Version:      "2.4.11",
+		DefaultValue: "134217728", // 128MB
```

Comment:
The default value is different from milvus.yaml, to correct that, you can delete the entries in milvus.yaml and re-generate it.

Reply:
done